pkg/handler/dashboard/dashboard.go

```diff
@@ -25,3 +29,29 @@ func New(store *store.Store, repo store.DBRepo, service *service.Service, logger
 		config:  cfg,
 	}
 }
+
+// ProjectSizes godoc
+// @Summary Get the total number of active member in each project
+// @Description Get the total number of active member in each project
+// @Tags Dashboard
+// @Accept json
+// @Produce json
+// @Param Authorization header string true "jwt token"
+// @Success 200 {object} view.ProjectSizeResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /dashboards/project-sizes [get]
```

Comment:
```suggestion
// @Router /dashboards/projects/sizes [get]
```
I think we should separate routes for each part in dashboard